rust/crates/langsmith-pyo3/README.md

```diff
@@ -23,3 +26,15 @@ To run install these bindings into another virtualenv (e.g. to run benchmarks),
 activate that virtualenv, then `cd` to this directory and run `uvx maturin develop --release`.
 When that command completes, the virtualenv will have an optimized build
 of `langsmith-pyo3` installed.
+
+### Testing
+
+Do not bare `cargo test`, *IT WILL NOT WORK*. You will get an inscrutable linker error like `undefined symbol: _Py_CheckFunctionResult` or `linker command failed with exit code 1`.
```

Comment:
```suggestion
Do not run `cargo test`, *IT WILL NOT WORK*. You will get an inscrutable linker error like `undefined symbol: _Py_CheckFunctionResult` or `linker command failed with exit code 1`.
```